games/d.yaml

```diff
@@ -396,7 +396,8 @@
 
 - name: Drugwars
   meta:
-    genre: [simulation]
+    genre: [Strategy, TBS]
```

Comment:
There is no point as cataloging a game as both Strategy and TBS. Also why isn't simulation appropriate here?